src/Arrays/array_operations.jl

```diff
@@ -92,3 +92,29 @@ A copy of the array where each negative entry is replaced by zero.
 function rectify(x::AbstractArray{N}) where {N<:Real}
     return map(xi -> max(xi, zero(N)), x)
 end
+
+"""
+    argmaxabs(x::AbstractArray)
+
+Return the index with the absolute-wise maximum entry.
+
+### Input
+
+- `x` -- array
+
+### Output
+
+The index `i` such that `|x[i]| >= |x[j]|` for all `j`.
+"""
+function argmaxabs(x::AbstractArray)
+    res = abs(first(x))
+    idx = 1
+    @inbounds for i in 2:length(x)
+        v = abs(x[i])
```

Comment:
The implementation assumes that `x`'s indexing starts from `1`, though it dispatches on `AbstractArray` hence using `eachindex` would be more robust. Since we are not doing this for other methods as well, I think it's good, but we could review it as some point. See https://docs.julialang.org/en/v1/manual/arrays/#Iteration

Reply:
You are right. I wanted to have a starting value in line 110 and then not check that entry twice, but I guess I can start with a dummy value instead.